python/cuml/cluster/CMakeLists.txt

```diff
@@ -0,0 +1,45 @@
+# =============================================================================
+# Copyright (c) 2022, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+# =============================================================================
+
+
+
+set(cython_sources
+    agglomerative.pyx
+    dbscan.pyx
+    hdbscan.pyx
+    kmeans.pyx
+)
+
+set(linked_libraries cuml::${CUML_CPP_TARGET})
```

Comment:
Should we just create a variable for `cuml::${CUML_CPP_TARGET}`? Otherwise you'll just keep repeating this everywhere.